packages/charts/src/utils/themes/theme_common.ts

```diff
@@ -6,28 +6,29 @@
  * Side Public License, v 1.
  */
 
+import { Color, Colors } from '../../common/colors';
 import { Margins } from '../dimensions';
 import { SharedGeometryStateStyle } from './theme';
 
-/** @public */
-export const DEFAULT_MISSING_COLOR = 'red';
+/** @internal */
+export const DEFAULT_MISSING_COLOR: Color = Colors.Red.keyword;
```

Comment:
the `Color.X.keyword` should already be typed as `Color`.

```suggestion
export const DEFAULT_MISSING_COLOR = Colors.Red.keyword;
```

Reply:
done